### ISSUE TITLE
AIR conditioner LG B09TS. The temperature sensor does not work and there is no ionizer

### ISSUE THREAD

**Issue by DivanX10** [ollo69/ha-smartthinq-sensors#159]
****
I checked the operation of the air conditioner in this integration and already have an understanding of what works and what does not.

1) The temperature sensor does not work. You can set the temperature, but it does not show the room temperature, all the time it shows the temperature of 25

2) No ionizer function

****

Проверил работу кондиционера в этой интеграции и уже есть понимание, что работает, а чего нет.
1) Не работает сенсор температуры. Можно выставить температуру, но не показывает комнатную температуру, все время показывает температуру 25
2) Нет функции ионизатор

****

Hass OS
Home Assistant 2021.5.5
Integration is installed SmartThinQ LGE Sensors v0.7.2
AIR conditioner LG B09TS

```
hvac_modes:
  - 'off'
  - heat
  - dry
  - cool
  - fan_only
  - heat_cool
min_temp: 18
max_temp: 30
target_temp_step: 1
fan_modes:
  - LOW
  - LOW_MID
  - MID
  - MID_HIGH
  - HIGH
  - NATURE
swing_modes:
  - 'OFF'
  - TOP
  - MIDDLE_TOP_1
  - MIDDLE_TOP_2
  - MIDDLE_BOTTOM_2
  - MIDDLE_BOTTOM_1
  - BOTTOM
  - SWING
current_temperature: 25.5
temperature: 22
fan_mode: LOW
swing_mode: BOTTOM
friendly_name: КОНДИЦИОНЕР
supported_features: 41
```

**Describe the bug**

**Expected behavior**
If applicable, a clear and concise description of what you expected to happen.

**Screenshots**

**Environment details:**
 - Environment (HASSIO, Raspbian, etc):
 - Home Assistant version installed:
 - Component version installed:
 - Last know working version:
 - LG device type and model with issue:
 - LG devices connected (list):

**Output of HA logs**
Paste the relavant output of the HA log here.

```

```

**Additional context**

**Comment by ollo69** [comment]
> 1\.  The temperature sensor does not work. You can set the temperature, but it does not show the room temperature, all the time it shows the temperature of 25

`Room` temperature is not `Target` temperature but `Current`. Try refresh your room...

> 2\. No ionizer function

Don't know what is ionizer, nor how it could be managed in HA

> Проверил работу кондиционера в этой интеграции и уже есть понимание, что работает, а чего нет.
> 
> 1. Не работает сенсор температуры. Можно выставить температуру, но не показывает комнатную температуру, все время показывает температуру 25
> 2. Нет функции ионизатор

Please only english here.

**Comment by DivanX10** [comment]
It was very interesting. As soon as I went to the LG SmartThinQ mobile app to take a screenshot, the room temperature was displayed correctly, and until I went to the mobile app, the temperature did not change. Maybe something is wrong with the synchronization?

**Comment by ollo69** [comment]
Just to understand, you are saying the room temperature sensor is not equal to the current temperature provided by the climate entity?

**Comment by DivanX10** [comment]
Until I went to the LG SmartThinQ mobile app, it was like this. This is one air conditioner, but works through two integrations. This is your integration and integration from [AlexxIT YandexStation](https://github.com/AlexxIT/YandexStation). To make it clear, I took a screenshot of a separate temperature sensor from xiaomi. In your integration, the room temperature sensor of the air conditioner is updated if I go to the mobile app, while the AlexxIT YandexStation integration shows the room temperature correctly and I do not need to go to the LG SmartThinQ mobile app

![image](https://user-images.githubusercontent.com/64090632/119034801-751e3600-b9b7-11eb-8c64-8c1715c88390.png)
![image](https://user-images.githubusercontent.com/64090632/119034726-54ee7700-b9b7-11eb-996d-5cdad32ad76c.png)
![image](https://user-images.githubusercontent.com/64090632/119034313-e6a9b480-b9b6-11eb-849b-f94acff6feb3.png)

**Comment by DivanX10** [comment]
Until I went to the LG SmartThinQ mobile app, it was like this. This is one air conditioner, but works through two integrations. This is your integration and integration from [AlexxIT YandexStation](https://github.com/AlexxIT/YandexStation). To make it clear, I took a screenshot of a separate temperature sensor from xiaomi. In your integration, the room temperature sensor of the air conditioner is updated if I go to the mobile app, while the AlexxIT YandexStation integration shows the room temperature correctly and I do not need to go to the LG SmartThinQ mobile app

![image](https://user-images.githubusercontent.com/64090632/119034801-751e3600-b9b7-11eb-8c64-8c1715c88390.png)
![image](https://user-images.githubusercontent.com/64090632/119034726-54ee7700-b9b7-11eb-996d-5cdad32ad76c.png)
![image](https://user-images.githubusercontent.com/64090632/119034313-e6a9b480-b9b6-11eb-849b-f94acff6feb3.png)

**Comment by ollo69** [comment]
Are you sure that AlexxIT integration is using current temperature from AC device. I'm not able to understand the integration features because read-me is not in English, but looking your images seems that `current_temperature` value come from 2 different sources. What do you mean when you say "the room temperature sensor of the air conditioner is updated if I go to the mobile app"? That when the mobile app is closed the `current_temperature` never change?

**Comment by DivanX10** [comment]
Yes, you got it right. If the LG SmartThinQ mobile app is closed, the `current temperature` data is not updated, it is necessary to go to the LG SmartThinQ mobile app, then there is synchronization and after synchronization, the `current temperature` in the Home Assistant starts showing the correct temperature, as in the LG SmartThinQ mobile app.

**Comment by ollo69** [comment]
This is odd. I would understand if also others have same problem, nobody else report this issue till now.

**Comment by DivanX10** [comment]
I recorded a video to show how it all works. Watch the video [here](https://drive.google.com/file/d/1AKN2MChWIA89yyJ8JOflQXa82X_J0et_/view?usp=sharing ).

**Comment by ollo69** [comment]
I see, Would like to understand if this is a common issue, I will ask in main ac thread,

**Comment by DivanX10** [comment]
Tell me, will it be possible to add the function of turning on and off the ionizer on the air conditioner? Can I provide some air conditioner data so that you can add it to the integration?

**Comment by qwer64** [comment]
У меня LG PC09SQ. Тоже проблема с синхронизацией текущей температуры. Показывает правильно после запуска LG Thinq

**Comment by xleonx90** [comment]
I join the problem of updating data on temperature and power consumption when the LG Thinq application is closed. As soon as the application is logged in, the data is updated and continues to be updated while the application is active. The problem is relevant only for an air conditioner with api v. 2.
I am also interested in the implementation of the inclusion of an ionizer.

Присоединяюсь к проблеме обновления данных о температуре и энергопотреблении при закрытом приложении LG Thinq. Как только осуществляется вход в приложение данные обновляются и продолжают обновляется пока приложение активно. Проблема актуальна только для кондиционера с api v.2.
Так же интересует реализация включения ионизатора.

**Comment by xleonx90** [comment]
For information on how the function works:
https://github.com/ollo69/ha-smartthinq-sensors/issues/37#issuecomment-847249781

**Comment by JezausTevas** [comment]
Has this issue been solved? I'm writting my own thinq2 api interpreter and found that "snapshot" data is not updated if the LG App is not open. There is even a parameter "timestamp" which gives UNIX timestamp of the time it was updated and it will not change after the LG App is closed.
It seems that there must be some API call to update or monitor the current state of the device that is executed upon LG App launch, but I am unable to find it.
If anyone has any information about this, please stand up
